modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -161,6 +161,130 @@ An additional role is required if the installation program is to create a vSpher
 |===
 ====
 
+.Roles and privileges required for installation in vCenter UI
+[%collapsible]
+====
+[cols="3a,3a,3a",options="header"]
+|===
+|vSphere object for role
+|When required
+|Required privileges in vCenter UI
```

Comment:
Change to "Required privileges in vCenter GUI"?

Reply:
Addressed